### DISCUSSION TITLE
Support CSS bundling (With @include CSS rules)

### DISCUSSION THREAD

**Discussion by trymeouteh** [rollup/rollup#5505] [Ideas]
Please consider supporting bundling CSS in Rollup.

By simply using an entry file (such as index.css, build.css), it will make a build from the entry file.

CSS files that have `@include 'my-style.css` will also grab the styles from *my-style.css* and add it to the bundle.

This will be very useful for web development.

**Comment by lukastaegert** [comment]
At the moment, CSS handling is not on the roadmap as it would increase the scope of Rollup core quite a bit, and it will likely not be on the roadmap for the years to come. Instead, I would invite people to write plugins to do that.
Or just use Vite, which uses Rollup under the hood and already injects the plugins to bundle CSS to my knowledge.
